data.js

```diff
@@ -1,4 +1,14 @@
 exports.projectsData = [
+    {
+    id: 206,
+    type: "video",
+    title: "Create Your Own ChatGPT in HTML CSS and JavaScript",
+    category: ["web-dev"],
+    tech: ["HTML","CSS","JavaScript", "OpenAI API", "API"],
+    datePublished: "July 12, 2023",
+    projectURL:"https://youtu.be/atKtG29iroY",
+
```

Comment:
Please include a difficulty level for this entry. This feature will be valuable for users when they start filtering by difficulty, a functionality that will soon be available on ProjectLearn.

Add a new key `difficulty` with one of these values: `easy`, `intermediate` or `advanced`